src/utils.js

```diff
@@ -78,6 +78,15 @@ export function getUpcomingDate(dateTimeList) {
   return sortDateTimeList(dateTimeList).filter((dateTimeList) => new Date(dateTimeList.date) >= new Date())[0];
 }
 
+export function isValidUrl(url){
```

Comment:
@urigat it would be better to check for `http` prefix as well:
```
function isValidHttpUrl(string) {
  let url;

  try {
    url = new URL(string);
  } catch (_) {
    return false;  
  }

  return url.protocol === "http:" || url.protocol === "https:";
}
```

See [reference](https://stackoverflow.com/a/43467144)

Reply:
Done.